### PR TITLE
add multiuser cluster import

### PR DESCRIPTION
**Pull request by mkudlej**

### CODE REVIEW

File: web/tendrl/multiuser_cluster_import.rst [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+
+.. test_action::
+   :step:
+       In first browsers click on *View Task Progress*.
+   :result:
+       Task in first browser should pass.
+
+.. test_action::
+   :step:
+       If import cluster task pass in second browser click on *Import* button.
```
Comment (ltrilety) [inline web/tendrl/multiuser_cluster_import.rst:52]: sorry, you lost me here, if it pass click on import? Like when import cluster pass, try it again?

File: web/tendrl/multiuser_cluster_import.rst [hunk 2]
```diff
@@ -1,2 +1,52 @@
...
+   :result:
+       There is page for new task opened with message: *Import Cluster Task Submitted*
+
+.. test_action::
+   :step:
+       In second browser click on *Vew Task Progress*.
```
Comment (ltrilety) [inline web/tendrl/multiuser_cluster_import.rst]: Vew -> View

File: web/tendrl/multiuser_cluster_import.rst [hunk 3]
```diff
@@ -1,2 +1,52 @@
...
+   :result:
+       Task insecond browser should fail.
```
Comment (ltrilety) [inline web/tendrl/multiuser_cluster_import.rst]: insecond -> in second

**Review by ltrilety — CHANGES_REQUESTED** [review]

File: web/tendrl/multiuser_cluster_import.rst [hunk 4]
```diff
@@ -1,2 +1,52 @@
...
+****************************
+
+:authors:
+          - mkudlej@redhat.com
+
+
+Description
+============
+
+Import cluster simultaneously from two browsers.
```
Comment (mbukatov) [inline web/tendrl/multiuser_cluster_import.rst]: This looks like the rare example of a test case, when a more deliberate explanation of the intent in the description section would be welcome.

File: web/tendrl/multiuser_cluster_import.rst [hunk 5]
```diff
@@ -1,2 +1,52 @@
-multiuser_cluster_import
-*******************************************************************************
\ No newline at end of file
+Multiuser - Cluster import
+****************************
+
+:authors:
+          - mkudlej@redhat.com
+
```
Comment (mbukatov) [inline web/tendrl/multiuser_cluster_import.rst]: Could you specify if this test case is positive or negative? Adding either `caseposneg: positive` or `caseposneg: positive`.

File: web/tendrl/multiuser_cluster_import.rst [hunk 6]
```diff
@@ -1,2 +1,52 @@
...
+Description
+============
+
+Import cluster simultaneously from two browsers.
+
+Setup
+======
+
+#. Prepare environment as described in :doc:`/configurations/multiuser_initial_setup`.
```
Comment (mbukatov) [inline web/tendrl/multiuser_cluster_import.rst:25]: Do you mean that you should log in with 2 different admin accounts, each in a separate browser?

File: web/tendrl/multiuser_cluster_import.rst (hunk 1 above)
Comment (mbukatov) [inline web/tendrl/multiuser_cluster_import.rst:52]: I would rephrase it as:

> When import cluster task started in 1st browser passes, click on Import* button in 2nd browser.

Feel free to tweak it based on your actual intentions.

File: web/tendrl/multiuser_cluster_import.rst (hunk 6 above)
Comment (mbukatov) [reply web/tendrl/multiuser_cluster_import.rst:25]: Would this test case be valid or the same if you log in with the same admin account from 2 different browsers?

**Review by mbukatov — CHANGES_REQUESTED** [review]
See my questions below, basically I'm asking to clarify intentions a bit.

File: web/tendrl/multiuser_cluster_import.rst (hunk 1 above)
Comment (mkudlej) [reply web/tendrl/multiuser_cluster_import.rst:52]: I've rewritten testcase description. Goal is to try to import cluster twice from 2 different browsers.

File: web/tendrl/multiuser_cluster_import.rst (hunk 6 above)
Comment (mkudlej) [reply web/tendrl/multiuser_cluster_import.rst:25]: Changed description.

File: web/tendrl/multiuser_cluster_import.rst (hunk 1 above)
Comment (mkudlej) [reply web/tendrl/multiuser_cluster_import.rst:52]: Changed description.

File: web/tendrl/multiuser_cluster_import.rst [hunk 7]
```diff
@@ -1,2 +1,55 @@
...
+
+.. test_action::
+   :step:
+       In first browsers click on *View Task Progress*.
+   :result:
+       Task in first browser should pass.
+
+.. test_action::
+   :step:
+       If import cluster task pass in second browser click on *Import* button.
```
Comment (ltrilety) [inline web/tendrl/multiuser_cluster_import.rst:52]: Maybe it's just my bad english, but I feel that the word order is bad.
I would rephrase it to something like:
> If import cluster task pass click on *Import* button in second browser.

Or
> Click on *Import* button in second browser if import cluster task pass (in the first one).

Hmm, probably even better will be to not use 'if import pass' but instead use 'when import pass'.

**Review by ltrilety — APPROVED** [review]
I have just slight note, nothing which we can't live with

File: web/tendrl/multiuser_cluster_import.rst [hunk 8]
```diff
@@ -1,2 +1,55 @@
...
+
+:authors:
+          - mkudlej@redhat.com
+:caseposneg: positive
+
+
+Description
+============
+
+Try to import cluster twice. Testcase is tested with one (or two users) and
```
Comment (mbukatov) [inline web/tendrl/multiuser_cluster_import.rst]: If "twice" means **two times simultaneously**, could we write it here?

File: web/tendrl/multiuser_cluster_import.rst [hunk 9]
```diff
@@ -1,2 +1,55 @@
...
+#. Prepare cluster ready for import.
+
+Test Steps
+===========
+
+.. test_action::
+   :step:
+       In both browsers click on *Import Cluster* button.
+   :result:
+       In both browsers *Cluster Import* wizard starts.
```
Comment (mbukatov) [inline web/tendrl/multiuser_cluster_import.rst:36]: Do you mean that you initiate import of the same cluster two times (almost at the same time)?

File: web/tendrl/multiuser_cluster_import.rst [hunk 10]
```diff
@@ -1,2 +1,55 @@
...
+
+.. test_action::
+   :step:
+       In first browser click on *Import* button.
```
Comment (mbukatov) [inline web/tendrl/multiuser_cluster_import.rst:40]: In a previous step, I just clicked on *Import Cluster* button. With that in mind, what do you mean by clicking on *import* button in only one browser?

**Review by mbukatov — CHANGES_REQUESTED** [review]
The description of the test steps is not very clear.

**Comment by mbukatov** [comment]
Clarification pending.

**Review by fbalak — APPROVED** [review]
LGTM

**Comment by mbukatov** [comment]
Clarification pending.
